cloud/cmd/cloudcore/cloudcore.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package main
 
 import (
+	"github.com/kubeedge/beehive/pkg/core"
+	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub"
 	"os"
 
 	"k8s.io/component-base/logs"
```

Comment:
Please sort the import. Those packages starting with `github.com/kubeedge` should be placed in the third part

Reply:
Thank you very much for your specific modification. I will modify and improve it as soon as possible.